### ISSUE TITLE
Datatable Error Trailing Data

### ISSUE THREAD

**Issue by miguel-piste** [thekordy/ticketit#546]
Hi guys, I just installed "Ticketit" on my server, everything works fine, except for the Ticket index, when I created the first ticket I got an error: "Exception Message: dataTrailing data", it is worth mentioning that I am using SQL Server 2012.
Can someone help me please?
![image](https://user-images.githubusercontent.com/58238107/71934655-85d75e80-316a-11ea-8ebb-dd7140669ac5.png)

**Comment by balping** [comment]
Hi,

Could you please check the network tab the developer console to see what is returned by the server?

**Comment by miguel-piste** [comment]
> Hi,
> 
> Could you please check the network tab the developer console to see what is returned by the server?

Of course, 
![image](https://user-images.githubusercontent.com/58238107/71935189-d13e3c80-316b-11ea-8be3-9a05387595d0.png)

and this
![image](https://user-images.githubusercontent.com/58238107/71935253-f8950980-316b-11ea-89de-977781842895.png)

**Comment by balping** [comment]
So it seems that the error message is coming from the server itself. Could you check the error log on the server too? Should be under storage/logs

**Comment by miguel-piste** [comment]
I did not understand correctly, check my server log?

**Comment by balping** [comment]
Yes please. The call stack might reveal what is causing the issue.

**Comment by miguel-piste** [comment]
Hi, I think it has to do with the date format.
![image](https://user-images.githubusercontent.com/58238107/71995549-625ef300-3200-11ea-9bb3-6105e661d24f.png)
Because the system recognizes that there are 3 tickets, but the table is where it does not load.
![image](https://user-images.githubusercontent.com/58238107/71995801-c08bd600-3200-11ea-9b74-83752f705f5f.png)

**Comment by miguel-piste** [comment]
Does anyone have any ideas to solve the error? please

**Comment by IamAhmedly** [comment]
I have the same issue and this what appears in the server log
![image](https://user-images.githubusercontent.com/27307416/72332326-e9154380-36ca-11ea-9dfd-cf419ab7cc16.png)

**Comment by balping** [comment]
@IamAhmedly Are you using SQL server as well?

**Comment by miguel-piste** [comment]
I have the same error that @IamAhmedly has in the log
![image](https://user-images.githubusercontent.com/58238107/72394200-f77b5380-36fa-11ea-9d1d-90ad1e4d5c61.png)

**Comment by IamAhmedly** [comment]
> @IamAhmedly Are you using SQL server as well?

I'm using MariaDB using Wamp server

**Comment by AhmedAlEssam** [comment]
This issue occurred when I generated demo content using this command,
`php artisan db:seed --class="Kordy\Ticketit\Seeds\TicketitTableSeeder"`
and the tickets don't appear in its' page 
according to this StackOverflow thread "[Laravel Carbon Data Missing](https://stackoverflow.com/questions/34968061/laravel-carbon-data-missing)" 

> This error arises when Carbon's `createFromFormat `function receives a date string that doesn't match the passed format string. More precisely this comes from the [DateTime::createFromFormat](https://www.php.net/manual/en/datetime.createfromformat.php) function

So I've opened the PHPMyAdmin to the tickets table and sorted the rows according to its' update_date searching for `Null` value, I've found about 6 rows with null value so I deleted the rows with null, and that fix the issue 

this should not happen in the first place 

UPDATE:
in the image the rows to be deleted ....not 6 rows ... about (24) rows open the image in new tab
i'm using laravel version 6 and i think the error caused by the lines from 81-92 in file path
\vendor\kordy\ticketit\src\Controllers\TicketsController.php 
![image](https://user-images.githubusercontent.com/18071971/72694804-53a10600-3b47-11ea-9b65-6449b3a55f98.png)

**Comment by miguel-piste** [comment]
In my case I solved it by converting the datetime fields to datetime2, I use Sql Server 2012
